migrations/versions/0383_update_default_templates.py

```diff
@@ -0,0 +1,55 @@
+"""
+
+Revision ID: 0383_update_default_templates.py
+Revises: 0382_remove_old_providers
+Create Date: 2023-01-10 11:42:25.633265
+
+"""
+import json
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+from flask import current_app
+
+revision = '0383_update_default_templates.py'
+down_revision = '0382_remove_old_providers'
+
+
+def upgrade():
```

Comment:
[question] is it possible to just use the commands function here?

Reply:
I'm not sure how compatible the different `SQLAlchemy` wrappers are, but I also think it's good for a migration to not depend on my utility function, if it doesn't have to.